components/molecule/photoUploader/src/index.js

```diff
@@ -516,6 +522,9 @@ MoleculePhotoUploader.propTypes = {
   /** Func to be executed when dropzone area is clicked */
   onEmptyViewClick: PropTypes.func,
 
+  /** Func to be executed when photo is being sorted */
+  onSortPhotoStart: PropTypes.func,
```

Comment:
🍆 not sure then what does `Start` mean there...? is it because the event fires when the user "starts" sorting photos?

Reply:
oh I see it's coupled to the name the ReactSortable component prop has... `onStart`

Reply:
still, I would keep it simple... `onSortPhotos` is my suggestion 😉

Reply:
Thanks!!!